cli/src/commands/new.rs

```diff
@@ -51,8 +51,9 @@ pub(crate) struct NewArgs {
     /// Parent(s) of the new change
     #[arg(default_value = "@")]
     pub(crate) revisions: Vec<RevisionArg>,
-    /// Ignored (but lets you pass `-r` for consistency with other commands)
-    #[arg(short = 'r', hide = true, action = clap::ArgAction::Count)]
+    /// Ignored (but lets you pass `-d`/`-r` for consistency with other
+    /// commands)
+    #[arg(short = 'd', hide = true, short_alias = 'r',  action = clap::ArgAction::Count)]
     unused_revision: u8,
```

Comment:
You could call this `unused_destination`.

Reply:
Done